hops/project.py

```diff
@@ -0,0 +1,82 @@
+"""
+A module for connecting to and working with Hopsworks projects.
+
+Using the utility functions you can connect to a project of a particular Hopsworks instance which sets up all the
+required environment variables and configuration parameters. Then you can use moduels such as `dataset` to interact
+with particular services of a project.
+"""
+
+import os
+import warnings
+
+from hops import util, constants
+from hops.exceptions import APIKeyFileNotFound
+
+
+def connect(project, host=None, port=443, scheme="https", hostname_verification=True,
```

Comment:
https://github.com/logicalclocks/hops-util-py/blob/9b2d2bf39118c93a244f76d1527175155900d403/hops/featurestore.py#L1876

In this method we also allow people to pass a pem file with the certificates in case they want to. I think we should have the same here.